From b64e5a7b7b9544d22946724b9a336385197ed517 Mon Sep 17 00:00:00 2001
From: Chmouel Boudjnah <chmouel@redhat.com>
Date: Tue, 31 Mar 2020 18:15:06 +0200
Subject: [PATCH] Skip duplications tests

They are flaky on OpenShift and runs most of the time in OOM.

Signed-off-by: Chmouel Boudjnah <chmouel@redhat.com>
---
 test/duplicate_test.go          | 1 +
 test/v1alpha1/duplicate_test.go | 1 +
 2 files changed, 2 insertions(+)

diff --git a/test/duplicate_test.go b/test/duplicate_test.go
index fd19ba9d..c2e24272 100644
--- a/test/duplicate_test.go
+++ b/test/duplicate_test.go
@@ -32,6 +32,7 @@ import (
 
 // TestDuplicatePodTaskRun creates 10 builds and checks that each of them has only one build pod.
 func TestDuplicatePodTaskRun(t *testing.T) {
+	t.Skip("This consumes too much memory on openshift and is buggy")
 	c, namespace := setup(t)
 	t.Parallel()
 
diff --git a/test/v1alpha1/duplicate_test.go b/test/v1alpha1/duplicate_test.go
index 083dbf5c..b302191a 100644
--- a/test/v1alpha1/duplicate_test.go
+++ b/test/v1alpha1/duplicate_test.go
@@ -31,6 +31,7 @@ import (
 
 // TestDuplicatePodTaskRun creates 10 builds and checks that each of them has only one build pod.
 func TestDuplicatePodTaskRun(t *testing.T) {
+	t.Skip("This consumes too much memory on openshift and is buggy")
 	c, namespace := setup(t)
 	t.Parallel()
 
-- 
2.25.0

